pkg/filters/processtree.go

```diff
@@ -17,10 +17,11 @@ type ProcessTreeFilter struct {
 	Enabled bool
 }
 
-func NewProcessTreeFilter() *ProcessTreeFilter {
+func NewProcessTreeFilter(mapName string) *ProcessTreeFilter {
 	return &ProcessTreeFilter{
 		PIDs:    map[uint32]bool{},
-		Enabled: false,
+		enabled: false,
```

Comment:
this field is not exposed now, but no API to get it is given, does this compile?

Reply:
fixed